src/Core/System/SalesChannel/Context/SalesChannelContextFactory.php

```diff
@@ -301,4 +308,20 @@ private function getCashRounding(BaseSalesChannelContext $context, ShippingLocat
 
         return [$context->getCurrency()->getItemRounding(), $context->getCurrency()->getTotalRounding()];
     }
+
+    private function getMeasurementInfo(BaseSalesChannelContext $context, ?string $domainId = null): MeasurementSystemInfo
```

Comment:
If i see it correctly this is not called 🤔

Reply:
forgot to remove this one actually 😅